packages/storybook/stories/QuickStarts/snippets/QuickstartCompositeAdapter.snippet.tsx

```diff
@@ -8,19 +8,32 @@ import {
 } from '@azure/communication-react';
 import React, { useEffect, useMemo, useState } from 'react';
 
+export type CompositeProps = {
+  userId: string; // '<Azure Communication Services Identifier>'
+  token: string; // '<Azure Communication Services Access Token>'
+  endpoint: string; // '<Azure Communication Services Resource Endpoint>'
+  displayName: string; // '<Display Name>'
+  groupId: string; // '<Developer generated GUID>'
```

Comment:
suggestion: rename to `locatorID` (to create commonality with AzureCommunicationCallAdapterArgs locator prop) or `callID` (to give better semantic meaning to what a groupID could mean)